sql/payday.sql

```diff
@@ -186,13 +186,14 @@ CREATE OR REPLACE FUNCTION process_payment_instruction() RETURNS trigger AS $$
               FROM payday_participants p
              WHERE username = NEW.participant
         );
+
         IF (NEW.amount + NEW.due <= participant.new_balance OR participant.card_hold_ok) THEN
             EXECUTE pay(NEW.participant, NEW.team, NEW.amount + NEW.due, 'to-team');
             RETURN NEW;
-        ELSE
+        ELSIF NEW.amount + NEW.due <= (SELECT minimum_charge FROM settings) THEN
```

Comment:
Yep, I was thinking about that too - and I now see one approach that'll give us the same end result, but might make more sense. How about not increase dues for any participant without [`has_credit_card`](https://github.com/gratipay/gratipay.com/blob/master/sql/payday.sql#L17) set to `true`?